docs/site/migration/auth/example.md

```diff
@@ -446,143 +446,64 @@ Try the 'team-member' role:
-Enter an empty property name when done
-
-? Enter the property name:
-   create src/models/user.model.ts
-   update src/models/index.ts
-
-Model User was created in src/models/
-```
-
-Create Team Model
-
-```sh
-$ lb4 model
-? Model class name: Team
-? Please select the model base class Entity (A persisted model with an ID)
-? Allow additional (free-form) properties? Yes
-Model Team1 will be created in src/models/team.model.ts
-
-Lets add a property to Team
-Enter an empty property name when done
-
-? Enter the property name: id
-? Property type: number
-? Is id the ID property? Yes
-? Is id generated automatically? No
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Team
-Enter an empty property name when done
-
-? Enter the property name: ownerId
-? Property type: number
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Team
-Enter an empty property name when done
-
-? Enter the property name: memberIds
-? Property type: array
-? Type of array items: number
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Team
-Enter an empty property name when done
-
-? Enter the property name:
-   create src/models/team.model.ts
-   update src/models/index.ts
-
-Model Team was created in src/models/
-```
-
-Create Project Model
-
-```sh
-$ lb4 model
-? Model class name: Project
-? Please select the model base class Entity (A persisted model with an ID)
-? Allow additional (free-form) properties? Yes
-Model Project1 will be created in src/models/project.model.ts
-
-Lets add a property to Project
-Enter an empty property name when done
-
-? Enter the property name: id
-? Property type: number
-? Is id the ID property? Yes
-? Is id generated automatically? No
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Project
-Enter an empty property name when done
-
-? Enter the property name: name
-? Property type: string
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Project
-Enter an empty property name when done
-
-? Enter the property name: balance
-? Property type: number
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Project
-Enter an empty property name when done
-
-? Enter the property name:
-   create src/models/project.model.ts
-   update src/models/index.ts
-
-Model Project was created in src/models/
+lb4 import-lb3-models <path_to_loopback-example-access-control> --outDir src/models
 ```
 
-Create UserCredentials
+Choose model `User`, `Project`, `Team` for the prompt.
+
+Considering the difference between the original application and the migrated
+one, you need to adjust the properties and add the relations.
+
+- For `Project`, open file `src/models/project.model.ts`
+
+  - decorate `ownerId` with `@belongsTo(() => User)` and remove the generated
```

Comment:
In our migration guide https://loopback.io/doc/en/lb4/migration-models-relations.html#migration-path, we recommend to run `lb4 relation` for adding relations. Could we follow that instructions too? Thanks.

Reply:
@dhmlau We don't expose endpoints for relations so the generated controllers are deleted, that's why I didn't user the cmd to generate relation. But refactored and added.

Reply:
Thanks. I didn't realize that.